jet-fuel/db/schema.rb

```diff
@@ -0,0 +1,12 @@
+ActiveRecord::Schema.define(version: 20150327064543) do
+
+  create_table "urls", force: :cascade do |t|
+    t.string   "full_url"
+    t.string   "short_url"
+    t.string   "title"
+    t.integer  "visit_count"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.